src/mlpack/tests/ann/init_rules_test.cpp

```diff
@@ -363,3 +363,35 @@ TEST_CASE("LecunNormalInitTest", "[InitRulesTest]")
   REQUIRE(weights3d.n_cols == cols);
   REQUIRE(weights3d.n_slices == slices);
 }
+
+/**
+ * Simple test of the LayerInitialization class.
+ */
+TEST_CASE("LayerInitTest", "[InitRulesTest]")
+{
+  Linear* linearModule_0 = new Linear(5);
+  Linear* linearModule_1 = new Linear(2);
```

Comment:
```suggestion
  Linear* linearModule0 = new Linear(5);
  Linear* linearModule1 = new Linear(2);
```

Also don't forget to `delete` if you are using `new`. :)  Honestly I don't think `new` is needed here, you could just use `Linear linearModule0(5);`.